src/test/java/org/junitpioneer/jupiter/DefaultLocaleTests.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2015-2018 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Locale;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("DefaultLocale extension")
+class DefaultLocaleTests {
```

Comment:
I don't know how to test the class level annotation since I see no way of knowing which locale has been the default locale before the class executed. Any idea?

Reply:
You can execute the test class via the `Launcher` API, checking the existing default locale both before and after the execution.

Reply:
@sormuras was talking about a dedicated API which might make it to JUnit 5.4. My plan was to add these tests once that has been released, but I can also try to come up with something based on your proposal. Will work on this on the weekend.

Reply:
These tests can be written like the ones in `TestIntegrationTests`.

Reply:
Even better, have a look at [`RepeatFailedTestTests`](https://github.com/junit-pioneer/junit-pioneer/pull/110/files#diff-94a60a248ed1fcd2fa7028a25ea6664e).